src/client/pages/OfferNew/Checkout/CheckoutContent.tsx

```diff
@@ -98,27 +90,23 @@ export const CheckoutContent: React.FC<Props> = ({
     <>
       <Section>
         <Excerpt>
-          <div>
-            {market === Market.Se && (
-              <InsuranceTypeLabel>
-                {textKeys.SIDEBAR_LABEL()}
-              </InsuranceTypeLabel>
-            )}
-            <InsuranceType>
+          <InsuranceHeading>
+            {market === Market.Se && <span>{textKeys.SIDEBAR_LABEL()}</span>}
+            <span>
               {!isBundle(offerData) &&
                 textKeys[
                   insuranceTypeTextKeys[offerData.quotes[0].contractType]
                 ]()}
               {isBundle(offerData) &&
                 textKeys.SIDEBAR_INSURANCE_TYPE_NO_BUNDLE()}
-            </InsuranceType>
-          </div>
+            </span>
+          </InsuranceHeading>
           <div>
             <Price
               loading={fakeLoading || reallyLoading}
               monthlyGross={offerData.cost.monthlyGross}
               monthlyNet={offerData.cost.monthlyNet}
-              monthlyCostDeduction={monthlyCostDeduction}
+              isDiscountPrice={monthlyCostDeduction}
```

Comment:
nitpick, but maybe we could rename the monthlyCostDeduction as well to something with "is" as postfix

Reply:
personally I think the name is fine, but is works as well. however the function to get that value on line 74 has that name already `const monthlyCostDeduction = isMonthlyCostDeduction(...`

Reply:
yeah I saw that too! As I think you are implying I agree,  it would be too much to change the function name because that is a rabbit whole that every pr should fix all naming convetions.  I just thought for this component that the variable could be named isDiscountPrice.